source/extensions/compression/common/BUILD

```diff
@@ -0,0 +1,19 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
```

Comment:
What is your opinion about this being in common/ (as you did here) vs having a compressor/decompressor sub-tree (like I did in my [PR](https://github.com/envoyproxy/envoy/pull/10744/files#diff-2cec098aba17a8c67877d0772b1167a7))?